<ui version="4.0" >
 <class>WelcomeWidgetClass</class>
 <widget class="QWidget" name="WelcomeWidgetClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>267</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <widget class="QFrame" name="frame_2" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>9</y>
     <width>370</width>
     <height>249</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth" >
    <number>0</number>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2" >
    <property name="spacing" >
     <number>6</number>
    </property>
    <property name="margin" >
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="topTitleFrame" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape" >
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth" >
       <number>0</number>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="titleLabel" >
         <property name="accessibleName" >
          <string>redLabel</string>
         </property>
         <property name="text" >
          <string>Title</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_5" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QFrame" name="frame" >
      <property name="frameShape" >
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth" >
       <number>0</number>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <property name="leftMargin" >
        <number>40</number>
       </property>
       <property name="rightMargin" >
        <number>40</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout" >
         <property name="leftMargin" >
          <number>9</number>
         </property>
         <property name="topMargin" >
          <number>9</number>
         </property>
         <property name="rightMargin" >
          <number>9</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <property name="verticalSpacing" >
          <number>0</number>
         </property>
         <item row="7" column="0" >
          <widget class="QLabel" name="findFriendsCommentLabel" >
           <property name="accessibleName" >
            <string>greyLabel</string>
           </property>
           <property name="text" >
            <string>FindFriendsText</string>
           </property>
           <property name="textFormat" >
            <enum>Qt::PlainText</enum>
           </property>
           <property name="alignment" >
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="wordWrap" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="profileCommentLabel" >
           <property name="accessibleName" >
            <string>greyLabel</string>
           </property>
           <property name="text" >
            <string>FillProfileText</string>
           </property>
           <property name="textFormat" >
            <enum>Qt::PlainText</enum>
           </property>
           <property name="alignment" >
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="wordWrap" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0" >
          <spacer name="verticalSpacer_3" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0" >
          <layout class="QHBoxLayout" name="horizontalLayout_3" >
           <property name="bottomMargin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="profileButton" >
             <property name="font" >
              <font>
               <family>MS Shell Dlg 2</family>
               <pointsize>8</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
               <underline>true</underline>
               <strikeout>false</strikeout>
              </font>
             </property>
             <property name="cursor" >
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="accessibleName" >
              <string>blueButton</string>
             </property>
             <property name="text" >
              <string>Profile</string>
             </property>
             <property name="flat" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3" >
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="5" column="0" >
          <layout class="QHBoxLayout" name="horizontalLayout_4" >
           <property name="bottomMargin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="findFriendsButton" >
             <property name="font" >
              <font>
               <family>MS Shell Dlg 2</family>
               <pointsize>8</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
               <underline>true</underline>
               <strikeout>false</strikeout>
              </font>
             </property>
             <property name="cursor" >
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="accessibleName" >
              <string>blueButton</string>
             </property>
             <property name="text" >
              <string>FindFriends</string>
             </property>
             <property name="flat" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4" >
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_4" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QFrame" name="bottomLinksFrame" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape" >
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth" >
       <number>0</number>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2" >
       <property name="spacing" >
        <number>30</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>87</width>
           <height>11</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="finishButton" >
         <property name="font" >
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>true</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="cursor" >
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="accessibleName" >
          <string>blueButton</string>
         </property>
         <property name="text" >
          <string>FinishButton</string>
         </property>
         <property name="default" >
          <bool>true</bool>
         </property>
         <property name="flat" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>10</width>
           <height>14</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="transBorder_4" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>2</height>
    </rect>
   </property>
   <property name="minimumSize" >
    <size>
     <width>0</width>
     <height>2</height>
    </size>
   </property>
   <property name="maximumSize" >
    <size>
     <width>16777215</width>
     <height>2</height>
    </size>
   </property>
   <property name="accessibleName" >
    <string>transBorder</string>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QFrame" name="transBorder" >
   <property name="geometry" >
    <rect>
     <x>386</x>
     <y>2</y>
     <width>2</width>
     <height>263</height>
    </rect>
   </property>
   <property name="minimumSize" >
    <size>
     <width>2</width>
     <height>263</height>
    </size>
   </property>
   <property name="maximumSize" >
    <size>
     <width>2</width>
     <height>263</height>
    </size>
   </property>
   <property name="accessibleName" >
    <string>transBorder</string>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth" >
    <number>1</number>
   </property>
  </widget>
  <widget class="QFrame" name="transBorder_2" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>2</y>
     <width>2</width>
     <height>263</height>
    </rect>
   </property>
   <property name="minimumSize" >
    <size>
     <width>2</width>
     <height>263</height>
    </size>
   </property>
   <property name="maximumSize" >
    <size>
     <width>2</width>
     <height>263</height>
    </size>
   </property>
   <property name="accessibleName" >
    <string>transBorder</string>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QFrame" name="transBorder_3" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>265</y>
     <width>401</width>
     <height>2</height>
    </rect>
   </property>
   <property name="minimumSize" >
    <size>
     <width>0</width>
     <height>2</height>
    </size>
   </property>
   <property name="maximumSize" >
    <size>
     <width>16777215</width>
     <height>2</height>
    </size>
   </property>
   <property name="accessibleName" >
    <string>transBorder</string>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="0" margin="0" />
 <resources/>
 <connections/>
</ui>
